backend/api/experiment.proto

```diff
@@ -114,7 +116,14 @@ message GetExperimentRequest {
 }
 
 message ListExperimentsRequest {
+  // A page token to request the next page of results. The token is acquried
+  // from the nextPageToken field of the response from the previous
+  // ListExperiment call.
```

Comment:
nit: Shall we also mention it is optional, when passing in an empty page token, we are fetching the first page?

Some wording suggestions:
```
For example, when you want to fetch page #2 of a list. Pass the token from the `nextPageToken` field of the page #1 ListExperiment response.
```

/cc @joeliedtke 
for review too

Reply:
Note, I think you can wait for a final agreement on the wording, before changing all the similar descriptions.

Reply:
I added " or is simply an empty string when fetching the first page."